src/lib/testutils.ts

```diff
@@ -33,6 +34,7 @@ export class TestLogger implements Logger {
     this.info = shouldLog ? createSpy(console.info) : createFake();
     this.verbose = shouldLog ? createSpy(console.log) : createFake();
     this.debug = createFake();
+    this.child = () => this;
   }
 }
 
```

Comment:
I had to stub the `child` method of logger because we use `TestLogger` in binary tests too. Should we move in under `src` directory? @willclarktech